solutions/devsprint-denis-vieira-1/app/src/main/java/com/devpass/githubapp/presentation/RepositoryDetailsHeaderFragment.kt

```diff
@@ -31,22 +32,35 @@ class RepositoryDetailsHeaderFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        with (binding) {
-            val stars = repository?.stargazersCount ?: 0
-            val forks = repository?.forksCount ?: 0
-
-            textviewRepositoryName.text = repository?.name
-            textviewRepositoryStars.text =
-                resources.getQuantityString(R.plurals.plural_name_repo_stars, stars, stars)
-            textviewRepositoryForks.text =
-                resources.getQuantityString(R.plurals.plural_name_repo_forks, forks, forks)
-
-            Glide
-                .with(root.context)
-                .load(repository?.owner?.avatarUrl)
-                .centerCrop()
-                .placeholder(R.drawable.ic_placeholder)
-                .into(imageviewRepository)
+        with(binding) {
```

Comment:
Aqui no onViewCreated você faz varias coisas diferentes que não estão diretamente relacionadas, o que acha de criar métodos para esse bloco interno separando por contexto ? 

é importante deixar esse métodos sobrescritos (overrided methods) apenas com as chamadas porque existe uma tendencia que eles fiquem sobrecarregados

Suggestion: 

```suggestion
      with(binding) {
         setupHeader(stars, forks)
         setupAvatarImage(avatarUrl)
         setupTabs(repo)
   }
```